app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,49 @@
+class AnswersController < ApplicationController
+
+  def new
+    @question = Question.find(params[:id])
+    @answer = @question.answers.new
+  end
+
+  def create
+    @question = Question.find(params[:question_id])
+    @answer = @question.answers.build(answer_params)
+    @answer.user_id = current_user.id
```

Comment:
It might be neater to add `.merge(user: current_user)` to the end of your permit/require chain in answer_params